terraform/gcp/modules/kms/kms.tf

```diff
@@ -53,3 +53,22 @@ resource "google_kms_key_ring_iam_member" "rekor_sa_kms_iam" {
   member      = format("serviceAccount:%s-rekor-sa@%s.iam.gserviceaccount.com", var.cluster_name, var.project_id)
   depends_on  = [google_kms_key_ring.rekor-keyring]
 }
+
+resource "google_kms_key_ring" "fulcio-keyring" {
+  name       = var.name
+  location   = var.location
+  project    = var.project_id
+  depends_on = [google_project_service.service]
+}
+
+resource "google_kms_crypto_key" "fulcio-intermediate-key" {
+  name     = var.key_name
```

Comment:
similarly `key_name` here is intended for the [rekor key name](https://github.com/sigstore/scaffolding/blob/main/terraform/gcp/modules/kms/variables.tf#L48-L51) (the naming is bad and should be called `rekor_key_name` for clarity)

Creating a `fulcio_key_name` variable in [variables.tf](https://github.com/sigstore/scaffolding/blob/main/terraform/gcp/modules/kms/variables.tf) should fix it.

Reply:
done